webpack.config.js

```diff
@@ -168,7 +158,36 @@ function getWebpackConfig( { cssFilename, externalizeWordPressPackages = false }
 								workers: Math.max( 2, Math.floor( os.cpus().length / 2 ) ),
 							},
 						},
-						babelLoader,
+						{
+							loader: 'babel-loader',
+							options: {
+								configFile: path.resolve( __dirname, 'babel.config.js' ),
+								babelrc: false,
+								cacheDirectory: path.join( __dirname, 'build', '.babel-client-cache' ),
+								cacheIdentifier,
+								overrides: [
```

Comment:
Just driving by, seeing that the `test` validation issue has been already solved (?), but I'm wondering: could the `overrides` field be in the `babel.config.js` rather than the webpack config? Other users of Babel in the repo (namely Jest tests) also want the `client/gutenberg/extensions` stuff to be correctly transpiled, don't they?